config/config.go

```diff
@@ -478,10 +473,6 @@ func New(validates ...Validate) (Config, error) {
 		}
 	}
 
-	// populdate chain ID and genesis timestamp
-	SetEVMNetworkID(cfg.Chain.EVMNetworkID)
-	SetGenesisTimestamp(cfg.Genesis.Timestamp)
```

Comment:
here cfg.Genesis is the default genesis, so it always use the same timestamp, resulting in same genesis block hash